data/hails.txt

```diff
@@ -431,11 +540,11 @@ phrase "friendly civilian"
 	word
 		" "
 	word
-		"song"
+		"band"
 	word
 		" "
 	phrase
-		"song"
+		"media name"
```

Comment:
If you're going to change this to be about bands rather than songs/media names, then this needs to be `band`, not `media name`.

Also some of the phrases that come next might need a bit of adjustment to refer to bands rather than songs

Reply:
@Pointedstick Finally getting a chance to address this. It looks like I got mixed up with the word `"song"` and the phrase `"song"`, and wound up making something nonsensical. I think also I got mixed up because some of the suffixes to the phrase were things like `"Check 'em out!"`, which sounded to me more like a band. A couple of ways I could address this:

1. Keep the word `"song"` and add quotes around the `"media name"`
  * E.g. "That song "Sad Archie" is so great! Have you heard it yet?"
  * E.g. "The new song "Nobody Gets out of Here Alive" is my kind of music! Check 'em out!"
2. Keep the word `"band"` and change the `"media name"` to the phrase `"band"`
  * E.g. "That band There Might Be Riots is so great! Have you heard it yet?"
  * E.g. "The new band "Mediocre Generica" is my kind of music! Check 'em out!"
3. Remove the word `"song"`/`"band"` and add quotes around the media name
  * E.g. "That "Sad Archie" is so great! Have you heard it yet?"
  * E.g. "The new "Nobody Gets out of Here Alive" is my kind of music! Check 'em out!"

Is there a preference?

Reply:
I'd say go with 1, since you've renamed the "song" phrase to "media name", so line 434/543 should really just stay as "song" instead of being changed to "band".